Description: The manual page incorrectly mentions that talloc is GPL, while it is LGPL
Author: Jelmer Vernooij <jelmer@samba.org>
Status: Cherry-picked from upstream
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=623879

--- a/talloc.3.xml	2011-07-25 13:30:22.301571046 +0200
+++ b/talloc.3.xml	2011-01-24 04:28:28.334518000 +0100
@@ -783,9 +783,9 @@
     </para>
     <para>
       This program is free software; you can redistribute it and/or modify
-      it under the terms of the GNU Lesser General Public License as 
-      published by the Free Software Foundation; either version 3 of the
-      License, or (at your option) any later version.
+      it under the terms of the GNU General Public License as published by
+      the Free Software Foundation; either version 3 of the License, or (at
+      your option) any later version.
     </para>
     <para>
       This program is distributed in the hope that it will be useful, but
